pkg/messaging/message.go

```diff
@@ -11,3 +11,22 @@ type Message struct {
 func (m *Message) IsBroadcast() bool {
 	return m.To == ""
 }
+
+type ControlPayload struct {
+	Type string `json:"type"`
+	Peer string `json:"peer"`
+}
+
+func NewPeerDisconnected(peerUID string) *ControlPayload {
```

Comment:
Maybe these should already return Message and any masrhalling should be done here?